source/content/drupal-8-configuration-management.md

```diff
@@ -4,10 +4,13 @@ description: Commit your Drupal configuration to version control.
 cms: "Drupal"
 categories: [develop]
 tags: [drush, workflow, webops]
+reviewed: "2020-10-21"
 ---
 Managing configuration is an extremely important part of any team website project, but in many cases, this area of the project does not receive as much attention as it deserves. The tools for Drupal 7 do not provide complete coverage of all configuration settings, leading to inconsistencies in configuration handling and inconvenient workarounds. This has led to configuration management becoming a real thorn in the side for many projects.
 
-Pantheon supports the [Drupal 8 Configuration Management system](https://www.drupal.org/documentation/administer/config) and defaults configuration into the `sites/default/config` directory for each Pantheon Drupal 8 site. You can export your configuration into that directory directly using Drush's config-export command or indirectly using Drupal's UI to download the configuration and then use SFTP/Git to place the configuration in `sites/default/config`. For more information on how this all works, check out Matt Cheney and David Strauss' presentation on [Drupal 8 CMI on Managed Workflow at Drupalcon Amsterdam](https://www.youtube.com/watch?v=eg2dtPFyGxs).
+Pantheon supports the [Drupal 8 Configuration Management system](https://www.drupal.org/documentation/administer/config) and defaults configuration into the `config` directory for each Pantheon Drupal 8.8+ site. Drupal versions before 8.8 store configurations in `sites/default/config`.
```

Comment:
Is this paragraph referring to sites created from the dashboard (drops-8)? Config is still `sites/default/config` there:

https://github.com/pantheon-systems/drops-8/blob/default/sites/default/settings.pantheon.php#L77

Reply:
@greg-1-anderson does that mean that the warning block [here](https://pantheon.io/docs/settings-php) is incorrect?

Reply:
Hey @greg-1-anderson! This request is based on this Warning message on this doc: https://pantheon.io/docs/settings-php. Is that information not correct?

Reply:
Note that the warning in that doc says at the bottom that this error is common in example-drops-8-composer. The example from the warning comes directly from example-drops-8-composer. The same sort of error can happen with drops-8 sites, if the user customizes the config location themselves. This is not going to be common. For users who do move their config directory on a drops-8 site, they by necessity must have used a different path, because the config path shown is from example-drops-8-composer and will not work on a drops-8 site.

Reply:
[Adjusted L11](https://github.com/pantheon-systems/documentation/pull/6075/commits/b2f5c89333dac68e22caf74a9895d515e4fefb79)